cmd/app-mesh-inject/main.go

```diff
@@ -62,6 +62,9 @@ func init() {
 	flag.BoolVar(&cfg.EnableJaegerTracing, "enable-jaeger-tracing", false, "Enable Envoy Jaeger tracing")
 	flag.StringVar(&cfg.JaegerAddress, "jaeger-address", "appmesh-jaeger.appmesh-system", "Jaeger address")
 	flag.StringVar(&cfg.JaegerPort, "jaeger-port", "9411", "Jaeger port")
+	flag.BoolVar(&cfg.EnableDatadogTracing, "enable-datadog-tracing", false, "Enable Envoy Datadog tracing")
```

Comment:
Since only one can be enabled at a time, should we change the flag format to be something like: 
```
--enable-tracing="jaeger"
--enable-tracing="datadog"
```
Perhaps this would only be valuable with some refactoring to more cleanly separate implementations from the core injector code, and that's probably not worth it at this point.  

Regardless, we should validate that both aren't enabled at once.

Reply:
That was my original thought indeed, I did not want to introduce a feature by deprecating flags that had just been added, but I think this is the cleanest way in my opinion.

I will do that now. 
Thanks, Nicholas.

Reply:
BTW don't forget x-ray tracing :). Unfortunately, that flag is named InjectXraySidecar and we cannot change that.